GVRf/Sample/simple-sample/src/org/gearvrf/simplesample/SampleViewManager.java

```diff
@@ -36,34 +57,194 @@ public void onInit(GVRContext gvrContext) {
         // in this sample
         mGVRContext = gvrContext;
 
-        GVRScene scene = gvrContext.getNextMainScene();
+        mMainScene = gvrContext.getNextMainScene();
 
         // set background color
-        GVRCameraRig mainCameraRig = scene.getMainCameraRig();
+        GVRCameraRig mainCameraRig = mMainScene.getMainCameraRig();
         mainCameraRig.getLeftCamera()
                 .setBackgroundColor(Color.WHITE);
         mainCameraRig.getRightCamera()
                 .setBackgroundColor(Color.WHITE);
+        
+        buildSceneObjects();
+        buildAnimation();
+        
+        buildEyePointeeHolder();
 
-        // load texture
-        GVRTexture texture = gvrContext.loadTexture(new GVRAndroidResource(
-                mGVRContext, R.drawable.gearvr_logo));
+        // add the scene object to the scene graph
+        mMainScene.addSceneObject(mMainObject);
+        
+    }
+    
+    private void buildSceneObjects(){
 
-        // create a scene object (this constructor creates a rectangular scene
-        // object that uses the standard 'unlit' shader)
-        GVRSceneObject sceneObject = new GVRSceneObject(gvrContext, 4.0f, 2.0f,
-                texture);
+        mMainObject = buildImageObject();
+//        mMainObject = buildVideoObject();
+//        mMainObject = buildWebViewObject();
+//        mMainObject = buildCameraObject();
+//        mMainObject = buildModelObject();
 
         // set the scene object position
-        sceneObject.getTransform().setPosition(0.0f, 0.0f, -3.0f);
-
-        // add the scene object to the scene graph
-        scene.addSceneObject(sceneObject);
-
+        mMainObject.getTransform().setPosition(0.0f, 0.0f, -5.0f);
     }
 
+    
+    
     @Override
     public void onStep() {
+    	checkEyePointee();
     }
 
+    
+    
+    // 1. Create image object
```

Comment:
I have a question of code formatter, is this right that use only 2 space instead of Tab ( 4 space )?
I mean..
usually, ( dot is space. )
void test ()
{
....test
}
but yours is,
void test()
{
..test
}